### PR TITLE
Update: medium color code issue fixed

### PR DESCRIPTION
**Pull request by ejaz-tapestry**
Ticket: https://jira.tapestry.support/browse/TORO-47150

### CODE REVIEW

File: src/brands/kate-spade/default/color/index.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
       "value": "#ffffff"
     },
     "neutral": {
+      "medium": {
+        "value": "#696969 "
```
Comment (p-musi) [inline src/brands/kate-spade/default/color/index.json:11]: I don't see #696969 referenced anywhere in the KSNA figma.

@ejaz-tapestry Are you sure this is the correct change?  In KSNA Figma, I can see color-neutral-medium is supposed to be "#575757".

Comment (p-musi) [reply src/brands/kate-spade/default/color/index.json:11]: We need to add color-neutral-dark as "#4A4A4A".

Comment (p-musi) [reply src/brands/kate-spade/default/color/index.json:11]: color-neutral-medium is supposed to be "#696969" only in KSJP, but you've added it here for KSNA.

Comment (ejaz-tapestry) [reply src/brands/kate-spade/default/color/index.json]: color added into KS NA is corrected and now code is added into KS JP file

**Review by aoverstreet-tapestry — APPROVED** [review]

**Review by aoverstreet-tapestry — APPROVED** [review]

**Pull request merged**